pages/index.tsx

```diff
@@ -0,0 +1,3 @@
+const HomePage = (): JSX.Element => <></>;
```

Comment:
This is mostly here to stop `next lint` from complaining about there being no `pages` directory